Makefile

```diff
@@ -109,6 +104,7 @@ install:
 .PHONY: remote-machine-install
 remote-machine-install:
 	cd src/sdk/pynni && python3 setup.py install $(PIP_MODE)
+	cd src/nni_manager/training_service_tool && python3 setup.py install $(PIP_MODE)
```

Comment:
i suggest move training_service_tool into training_service, for example, src/nni_manager/training_service/tools